shared/workspace_repo/api_client.go

```diff
@@ -106,6 +106,11 @@ func (c *WorkspaceRepo) getTfc(ctx context.Context) (*tfe.Client, error) {
 	return c.tfc, nil
 }
 
+func (c *WorkspaceRepo) validateAccess(ctx context.Context) error {
+	_, err := c.tfc.Organizations.List(ctx, &tfe.OrganizationListOptions{})
```

Comment:
Does this throw an error when the token is invalid? Looking at the source code for this library, it seems like it only returns the errors of `req.Do`  https://github.com/hashicorp/go-tfe/blob/b5d742f97b1ba8f474da7d467df1566c2aee4852/organization.go#LL254C13-L254C13. According to the docs, req.Do doesn't return an error for non-200 HTTP responses.

Reply:
It will throw an error if the token is missing or invalid -- went through pretty thorough testing.